src/files-card-body.tsx

```diff
@@ -273,14 +274,38 @@ export const FilesCardBody = ({
             }
         };
 
-        const hasNoKeydownModifiers = (event: KeyboardEvent) => {
+        const hasNoModifiers = (event: KeyboardEvent) => {
             return !event.shiftKey && !event.ctrlKey && !event.altKey && !event.metaKey;
         };
 
+        const dirNavigationModifiers = (e: KeyboardEvent) => {
+            if (isApple) {
+                // use both meta and alt key on apple devices to support external non-apple keyboards
+                return !e.ctrlKey && !e.shiftKey && (e.altKey || e.metaKey);
+            } else {
+                return !e.ctrlKey && !e.shiftKey && e.altKey && !e.metaKey;
+            }
+        };
+
+        // Modifiers used for copy, paste and select all shortcuts
+        const editingModifiers = (e: KeyboardEvent) => {
+            // Keep standard text editing behavior by excluding input fields
+            if ((e.target instanceof HTMLInputElement)) {
+                return false;
+            }
+
+            if (isApple) {
+                // use both meta and ctrl key on apple devices to support external non-apple keyboards
+                return !e.shiftKey && !e.altKey && (e.ctrlKey || e.metaKey);
+            } else {
+                return !e.shiftKey && !e.altKey && e.ctrlKey && !e.metaKey;
+            }
+        };
+
         const onKeyboardNav = (e: KeyboardEvent) => {
             switch (e.key) {
             case "ArrowRight":
-                if (hasNoKeydownModifiers(e)) {
+                if (hasNoModifiers(e) && sortedFiles.length > 0) {
```

Comment:
`sortedFiles.length > 0` is new, why do we add this here. It doesn't feel Mac specific but it however does make sense :)

Reply:
Not mac specific. I managed to run into `oops` when testing keybinds and figured this should be added.